### PR TITLE
pilot: support tls

### PR DESCRIPTION
**Pull request by chai2010**

### CODE REVIEW

**Comment by chai2010** [comment]
ping

File: api/metadata/pilot/pilot.proto [hunk 1]
```diff
@@ -52,6 +50,11 @@ service PilotService {
 
 	rpc RunCommandOnFrontgateNode (metadata.types.RunCommandOnFrontgateRequest) returns (metadata.types.String);
 	rpc RunCommandOnDrone (metadata.types.RunCommandOnDroneRequest) returns (metadata.types.String);
+}
 
+service PilotServiceForFG {
```
Comment (chilianyi) [inline api/metadata/pilot/pilot.proto]: It's not necessary to abbreviate, PilotServiceForFrontgate is ok.

File: api/metadata/types/pilot.proto [hunk 2]
```diff
@@ -13,6 +13,7 @@ message PilotConfig {
 	string host = 2;
 	int32 listen_port = 4;
 	string log_level = 5;
+	int32 for_fg_listen_port = 6;
```
Comment (chilianyi) [inline api/metadata/types/pilot.proto]: ditto

File: metadata/cmd/pilot/Makefile [hunk 3]
```diff
@@ -73,6 +73,7 @@ confd-stop: $(TARG).out
 clean: stop
 	-rm $(TARG).out
 	-rm $(TARG).out.log
+	- rm *.key *.pem
```
Comment (chilianyi) [inline metadata/cmd/pilot/Makefile]: fmt

File: api/metadata/pilot/pilot.proto (hunk 1 above)
Comment (chai2010) [reply api/metadata/pilot/pilot.proto]: fixed

File: api/metadata/types/pilot.proto (hunk 2 above)
Comment (chai2010) [reply api/metadata/types/pilot.proto]: fixed

File: metadata/cmd/pilot/Makefile (hunk 3 above)
Comment (chai2010) [reply metadata/cmd/pilot/Makefile]: fixed

**Pull request merged**